components/dailyList/dailyList.js

```diff
@@ -0,0 +1,53 @@
+import { createElement, formatDateToKorean } from '../../utils.js';
+import createDaily from './daily.js';
+
+export default function createDaliyList(dailyInfo) {
+    const { date } = dailyInfo;
+    const dateToKorean = formatDateToKorean(date);
+
+    const dailyListContainerInnerHtml = `
+        <div class="daily-header">
+            <div class="date-info">${dateToKorean}</div>
+        </div>
+        <ol class="daliy-line-wrapper"></ol>
+    `;
+
+    const $dailyListContainer = createElement(
+        'li',
+        {
+            class: 'day-container',
+        },
+        dailyListContainerInnerHtml,
+    );
+
+    const $dailyListWrapper = $dailyListContainer.querySelector(
+        '.daliy-line-wrapper',
+    );
+
+    let totalIncome = 0;
+    let totalExpense = 0;
+
+    dailyInfo.items.forEach((info) => {
+        $dailyListWrapper.appendChild(createDaily(info));
+        console.log(info);
+        if (info.amount > 0) {
+            totalIncome += info.amount;
+        } else {
+            totalExpense -= info.amount;
+        }
+    });
+
+    let amountHtml = '';
+    amountHtml += totalIncome != 0 ? `<div>수입 ${totalIncome}<div>` : '';
```

Comment:
닫힘 태그가 <div>로 되어있어요!

```suggestion
    amountHtml += totalIncome != 0 ? `<div>수입 ${totalIncome}</div>` : '';
```

Reply:
헙. 감사합니다. 자세히 봐주셨군요